keyboards/mokey/mokey12x2/info.json

```
@@ -0,0 +1,48 @@
+{
+    "keyboard_name": "Mokey12x2",
+    "manufacturer": "Mokey",
+    "url": "https://github.com/rhmokey",
+    "maintainer": "Mokey",
+    "processor": "STM32F103",
+    "bootloader": "stm32-dfu",
```

```suggestion
    "development_board": "bluepill",
```
If this is a Bluepill board, it should have the STM32Duino bootloader, not stm32-dfu, no?

---

> 如果这是 Bluepill 板，它应该有 STM32Duino 引导加载程序，而不是 stm32-dfu，不是吗？

Problem solved